dashboard/app/controllers/api/v1/pd/workshop_enrollments_controller.rb

```diff
@@ -102,6 +102,16 @@ def cancel
     Pd::WorkshopMailer.organizer_cancel_receipt(enrollment).deliver_now
   end
 
+  # POST /api/v1/pd/workshops/1/enrollments/move
+  def move
```

Comment:
It wasn't intuitive to me at first that this moves the listed enrollments _to_ the specified workshop - might be worth a descriptive comment on this controller method.

Reply:
Oh wait... now I realize that's not what happens either.  What's the point of using the source workshop in the query here?  It should be fast to look up the enrollments by id.